spec/lib/tasks/scripts/add_permission_to_access_level_spec.rb

```diff
@@ -1,191 +0,0 @@
-require "rails_helper"
-require "tasks/scripts/add_permission_to_access_level"
-
-RSpec.describe AddPermissionToAccessLevel do
```

Comment:
Deleted these specs since the User model traits used to set up the old-permissions users are now gone.
The actual task is still around and used in the dev-env data migration, that will go away in a future PR